app/lib/foreman_git_templates/renderer/source/repository.rb

```diff
@@ -5,6 +5,7 @@ module Renderer
     module Source
       class Repository < Foreman::Renderer::Source::Base
         def initialize(template, repository_path)
+          super(template)
           @template = template
```

Comment:
The parent already does this, so it could be dropped now that you invoke `super`?
```suggestion
```